_posts/en/newsletters/2023-03-08-newsletter.md

```diff
@@ -0,0 +1,209 @@
+      timelock has expired and then spending the unvaulting output,
+      choosing the tapleaf with the `OP_FORWARD_OUTPUTS` opcode.  The
+      opcode verifies that the spending transaction's output amounts and
+      scripts hash to the same commitment Alice made in the previous
+      transaction.  In this case, Alice has successfully deposited funds
+      to a vault, begun an unvaulting, been forced to wait at least
+      1,000 blocks to allow her monitoring programs to verify she really
+      did want to spend the funds to the specified outputs, and
+      completed the spend.
+
+    - If something goes wrong, *Alice freezes the funds*.  She can do
+      this at any time up until an unvaulting is completed by simply
+      choosing to spend using the freeze tapscript from the output of
+      either the vaulting or initial unvaulting transactions.  Recall
+      that Alice explicitly placed the freeze tapscript in the vaulting
+      transaction, and note that it was implicitly carried over by the
+      transaction with initiated the unvaulting.
+
+  One of the advantages to users of this approach over the original
+  `OP_VAULT` design is that the freeze tapscript can contain any
+  authorization conditions Alice wants to specify.  In the `OP_VAULT`
+  proposal, anyone knowing the parameters chosen by Alice could spend
+  her funds to the freeze script.  That wasn't a security problem but it
+  could be annoying.  In Sanders's design, Alice could (for example)
+  require a signature from a very lightly protected wallet in order to
+  initiate a freeze---this would perhaps be enough of a burden to
+  prevent most griefing attacks but not enough of a barrier to prevent
+  Alice from quickly freezing her funds in an emergency.
+
+  Several other advantages are aimed at making the consensus-enforced
+  [vaulting protocol][topic vaults] easier to understand and verify as
+  safe.
+
+  As of this writing, Sanders's proposal had only received limited
+  discussion on the mailing list.
+
+- **New Optech Podcast:** the weekly Optech Audio Recap hosted on
+  Twitter Spaces is now available as a podcast.  Each episode will be
+  available on all popular podcast platforms and on the Optech website
+  as a transcript.  For more details, including why we think this is a
+  major step forward in Optech's mission to improve Bitcoin technical
+  communication, please see our [blog post][podcast post].
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[bitcoin-inquisition: Activation logic for testing consensus changes][review club bi-16]
+is a PR by Anthony Towns that improves activation and deactivation
+logic for the [Bitcoin Inquisition][] project, which runs on [signet][topic signet].
```

Comment:
I quite strongly think "improves" is not the right word here. It swaps it out for a different mechanism; they are used in different contexts with very different goals. I don't think there is any notion of one being better than the other.
```suggestion
is a PR by Anthony Towns that adds a new method for activating and deactivating
soft forks in the [Bitcoin Inquisition][] project, designed to be run on [signet][topic signet]
and used for testing.
```